crates/mzoon/src/main.rs

```diff
@@ -320,6 +320,7 @@ fn start_backend_watcher(
 }
 
 fn generate_backend_build_id() {
+    fs::create_dir_all("backend/private").unwrap();
```

Comment:
Please remove this line and add a `private` directory with `.gitignore` to the `svg` example. Build id, certificates and other stuff is stored in `private` and you don't want to accidentally push it to Git (for security reasons, unnecessary commit changes, incompatible certs among envs, etc.)

[private](https://github.com/MoonZoon/MoonZoon/tree/main/examples/counter/backend/private) in `counter` example